test/transferSpec.js

```diff
@@ -41,31 +41,15 @@ describe('Plugin transfers (optimistic)', function () {
       assert.isFunction(this.pluginA.send)
     })
 
-    it('should send an optimistic transfer with 0 amount', function (done) {
```

Comment:
https://github.com/interledger/rfcs/blob/master/0004-ledger-plugin-interface/0004-ledger-plugin-interface.md#ledger-transfers says that "all plugins MUST implement 0-amount transfers." It does seem impractical for now though. maybe skip this instead of deleting it?